app/templates/partials/question.html

```diff
@@ -24,13 +24,45 @@ <h3 class="question__subtitle mars">{{question_subtitle}}</h3>
 {% endset %}
 
 {% set question_answers %}
-  <div class="question__answers">
-    {%- for answer in question.answers -%}
-      {% include theme('partials/answer.html') %}
-    {%- endfor -%}
+ <div class="question__answers">
+   {%- if question.schema_item.type == 'RepeatingAnswer' %}
+     {%- for answer in question.answers %}
+       <div class="question__answer js-household-person">
+         <h3 class="neptune">Person <span class="js-household-loopindex">{{ loop.index }}</span>
```

Comment:
Even better, any way this could come from the Schema (probably one for a later PR)?